.gitignore

```diff
@@ -1,5 +1,4 @@
 vendor/
 test.php
-phpunit.xml
```

Comment:
phpunit.xml should be in the .gitignore, using phpunit.xml.dist as the committed filename instead. This allows phpunit.xml.dist to be copied to phpunit.xml for local changes without getting accidentally committed. PHPUnit will read phpunit.xml.dist if phpunit.xml doesn't exist.